munge/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "dependencies": {
+    "@types/chai": "^4.2.19",
+    "@types/lodash": "^4.14.170",
+    "@types/mocha": "^8.2.2",
+    "commander": "^8.0.0",
+    "execifmain": "^0.0.2",
+    "hfstol": "^0.0.3",
+    "prettier": "^2.3.2",
+    "sucrase": "^3.19.0"
+  },
+  "devDependencies": {
+    "@types/node": "^15.14.0",
+    "@types/prettier": "^2.3.1",
+    "chai": "^4.3.4",
+    "mocha": "^9.0.1",
+    "typescript": "^4.3.5"
+  }
```

Comment:
What's the basis for distinguishing prod vs. dev dependencies for this package? I would have assumed they'd all be dev dependencies, since none of them get used in production code.

Reply:
For this sort of command-line stuff, the distinction I try to use is dev-stuff/build-time stuff, vs. stuff needed to run the end product. So `typescript` and `mocha` and all the `@types/` stuff would be dev dependencies, while `commander` is a normal dependency since it’s used at run time.

This can be useful when the transpiled code is deployed to a server to run as a backend. Then you save some time and disk space on the install, and may have a smaller security footprint as well.

I realize I haven’t actually followed that well in this particular `package.json`; I will clean that up.